test/utility/misc.c

```diff
@@ -257,6 +257,21 @@ testutil_copy_data(const char *dir)
         testutil_die(status, "system: %s", buf);
 }
 
+/*
+ * testutil_clean_backup_data --
+ *     Delete the backed up data saved in testutil_copy_data.
+ */
+void
+testutil_clean_backup_data(const char *dir)
+{
+    int status;
+    char buf[512];
+
+    testutil_check(__wt_snprintf(buf, sizeof(buf), "rm -rf ../%s.SAVE", dir));
```

Comment:
is it reasonably possible for `dir` to contain a space/asterisk/other unexpected thing? if so, `buf` could end up as "rm -rf ../ whatever.SAVE" which would be bad.

Reply:
Yep, very good point. I think we're depending on valid results in sane. This probably comes from a call to `pwd` but I'll check.

Reply:
I think we're ok with this. `dir` is either a constant (`WT_TEST`) or passed in via the `-h` flag. 
bash is evaluating this input when we call the test so we can't accidentally glob and spaces are interpreted as a second argument. The only fail case is if someone passes in `-h file\ name` which will become a space.

Reply:
ok. given we're not using it interactively (from the above conversation) that seems fine.